### ISSUE TITLE
Can't direct upload to DreamObjects (S3 compatible OpenStack installation)

### ISSUE THREAD

**Issue by andyjeffries** [tors/jquery-fileupload-rails#92]
When I try to do an upload, it takes a while uploading the file (e.g. I've got over the CORS hurdle), but after finishing uploading the file the server returns:

```
<?xml version="1.0" encoding="UTF-8"?><Error><Code>InvalidArgument</Code><Message>Missing access key</Message></Error>
```

There's no access key in the form-data, but I didn't think there was supposed to be (I thought that was the whole idea of the presigned post).

I can share any of the code I've written, request information, etc - just let me know what's needed (I don't want to share it all by default in case there's something I need to keep secure, I'm new to this direct-s3-upload functionality)

**Comment by medBouzid** [comment]
have you found any solution ?

**Comment by andyjeffries** [comment]
No, I think my only solution is to just use S3... But I've ignored this problem in my project up until now, hoping that the author will post a solution.

**Comment by felixbuenemann** [comment]
The author of jQuery File Upload is [Sebastian Tschan](https://github.com/blueimp) and the repo is at [blueimp/jQuery-File-Upload](https://github.com/blueimp/jQuery-File-Upload).

This repo merely repackages the assets as a ruby gem and adds some middleware to support iFrame uploads for ancient browsers.
